images/newrelic/configs/main.tf

```diff
@@ -0,0 +1,66 @@
+terraform {
+  required_providers {
+    apko = { source = "chainguard-dev/apko" }
+  }
+}
+
+locals {
+  components = {
+    "fluent-bit-output" = {
+      main_package   = ["newrelic-fluent-bit-output${var.suffix}"]
```

Comment:
I think we want this to just be a string, otherwise the output is a list, otherwise this LGTM.

Reply:
absolutely!